### PR TITLE
Add Typeform Navigator.

### PR DESCRIPTION
**Pull request by jdvr**
## Description

New extension to browse your Typeform account and check forms stats.

## Screencast

https://www.youtube.com/watch?v=3yBbrCWXLeg

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/typeform-navigator/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  );
+}
+
+function FormActions({ form }: { form: FormOverview }) {
+  return (
+    <>
+      <Action.OpenInBrowser title="Check results" url={`${ADMIN_FORM_BASE_URL}${form.id}/results`} />
+      <Action.OpenInBrowser title="Fill it" url={form._links.display} />
+      <Action.CopyToClipboard
+        title="Copy link"
```
Comment (pernielsentikaer) [inline extensions/typeform-navigator/src/index.tsx:194]:
```suggestion
        title="Copy Link"
```

File: extensions/typeform-navigator/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+- Grab the link into clipboard using \`cmd+.\`
+      `}
+    />
+  );
+}
+
+function FormActions({ form }: { form: FormOverview }) {
+  return (
+    <>
+      <Action.OpenInBrowser title="Check results" url={`${ADMIN_FORM_BASE_URL}${form.id}/results`} />
```
Comment (pernielsentikaer) [inline extensions/typeform-navigator/src/index.tsx:191]:
```suggestion
      <Action.OpenInBrowser title="Check Results" url={`${ADMIN_FORM_BASE_URL}${form.id}/results`} />
```

File: extensions/typeform-navigator/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+      <Action.OpenInBrowser title="Fill it" url={form._links.display} />
```
Comment (pernielsentikaer) [inline extensions/typeform-navigator/src/index.tsx:192]:
```suggestion
      <Action.OpenInBrowser title="Fill Form" url={form._links.display} />
```

File: extensions/typeform-navigator/package.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  "categories": [
+    "Web",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Typeform Navigator",
+      "subtitle": "Check your typeform workspaces and forms",
```
Comment (pernielsentikaer) [inline extensions/typeform-navigator/package.json:17]:
```suggestion
```

File: extensions/typeform-navigator/src/dto/index.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  default: boolean;
+  forms: {
+    count: number;
+    href: string;
+  };
+  id: string;
+  name: string;
+  account_id: string;
+  self: {
+    href: "https://api.typeform.com/workspaces/8qUjTK";
```
Comment (pernielsentikaer) [inline extensions/typeform-navigator/src/dto/index.ts:11]: Is this supposed to be like this?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @jdvr 👋 

Thanks for an awesome extension 🔥 

I have some feedback and suggestions for you to address 😄 

- If I just input a random token, it says "No Results" - It would be excellent with a [failure toast](https://developers.raycast.com/api-reference/feedback/toast#api-reference) telling about it

- If no lists are added to Typeform, it just dies. Could you handle this gracefully instead?

CleanShot 2022-04-19 at 08 01 09@2x


- It would be AWESOME if you added the step "Workspaces" with our [dropdown component](https://developers.raycast.com/api-reference/user-interface/list#list.dropdown) 

CleanShot 2022-04-19 at 08 03 08@2x

Looking forward to testing this again! 🚀 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @jdvr 👋 

Thanks for a wonderful extension! 🔥 

I tested it out and now it works as expected 💪

**Comment by jdvr** [comment]
Thanks @pernielsentikaer

**Pull request merged**